src/components/Header.tsx

```diff
@@ -6,7 +6,7 @@ import { setArticlesAuthor } from "redux/slices/articlesSlice"
 
 function Header() {
   const dispatch = useDispatch()
-  const { user } = useSelector((state) => state)
+  const { user }: any = useSelector((state) => state)
```

Comment:
Tutaj jak wyzej, jak dodasz typy do store i do rootReducer to tu nie powinno juz krzyczec bez `any`